Pipfile

```diff
@@ -40,6 +39,7 @@ mistune = "==0.8.4"
 myst-parser = "*"
 pre-commit = "*"
 bandit = "*"
+httpx = "*"
```

Comment:
Nit (to remind us in the future why we need it as a dev-package):

```suggestion
# Required to use `TestClient` from `fastapi`
httpx
```

Reply:
I didn't add it because it fails straightforwardly in the TestClient import.

Reply:
I agree it will be useful to add this comment.
I don't insist on this as well.

Reply:
> I didn't add it because it fails straightforwardly in the TestClient import.

In that case, I see your point.

> It sounds like we should comment why we use all other dependencies rofl

Hahaha no that is not necessary :laughing: Jokes aside, I was mentioning this because I remember an issue that we had some time ago where I had to check the `Pipfile` requirements one by one because extra requirements had been accidentally added. So basically what I did was to check which ones were being imported in our source code (`from <package-name> import x` or `import <package-name>`). In this case I would have probably accidentally removed `httpx` from the `Pipfile`.
But indeed, if you run the tests as you mentioned it would have probably be an easy fix to see that `httpx` would be missing.

Reply:
Great argumentation @KAUTH.